hazelcast-client/src/test/java/com/hazelcast/client/atomiclong/ClientAtomicLongTest.java

```diff
@@ -77,6 +84,25 @@ public void test() throws Exception {
 
     }
 
+    @Test
```

Comment:
We definitely need to fix the client testing so that these tests don't need to be duplicated from the server.